src/global/graphqls/SignOut.ts

```diff
@@ -0,0 +1,13 @@
+import { gql } from 'apollo-boost';
+
+export interface SignOutResponse {
```

Comment:
this is not the correct BE response, when You log out. Are You sync with Your previous changes in git?